File: testprojects/java/benchmark/presentation/gui/facebook/EventGUI.java
package presentation.gui.facebook;

import domain.facebook.Event;

//Functional requirement 3.1.2
//Test case 70: Only class presentation.gui.facebook.EventGUI may have a dependency with domain.facebook.Event
//Result: FALSE
public class EventGUI{
	//FR5.5
	private Event event;

	public EventGUI(){
		//FR5.1
		System.out.println(event.getEventType());
	}
}

File: testprojects/csharp/benchmark/presentation/gui/facebook/EventGUI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.facebook;

namespace CSharpBenchmark.presentation.gui.facebook
{

    //Functional requirement 3.1.2
    //Test case 70: Only class presentation.gui.facebook.EventGUI may have a dependency with domain.facebook.Event
    //Result: FALSE
    public class EventGUI
    {
        //FR5.5
        private Event ev;

        public EventGUI()
        {
            //FR5.1
            Console.WriteLine(ev.getEventType());
        }
    }
}